.github/workflows/ci.yml

```diff
@@ -95,8 +95,9 @@ jobs:
         fi
     - name: Install dependencies
       run: |
+        python -m pip install --upgrade pip
         pip install pytest-cov
-        pip install .
+        pip install .[ezancestry]
```

Comment:
@apriha I don't understand what's going on here. I tried this in a docker container from `ubuntu:latest` locally and it works just fine.

Reply:
Hi @arvkevi , turns out the Python 3.7 job is failing due to `ezancestry` requiring Python 3.8+, so the other jobs get cancelled.

However, I think it might be cleaner to mock `ezancestry` for test purposes vs. testing all permutations of it installed or not. I was going to work on that as soon as I can find some time. Thoughts?